### ISSUE TITLE
PFN_vkCreateWin32SurfaceKHR not defined error on windows

### ISSUE THREAD

**Issue by griffin2000** [realitix/vulkan#5]
I get the following error running the example on Windows. It looks like the function vkCreateWin32SurfaceKHR is defined in the DLL but the type 'PFN_vkCreateWin32SurfaceKHR is not.

```

Create windows surface
Traceback (most recent call last):
  File "example.py", line 145, in <module>
    surface = surface_mapping[wm_info.subsystem]()
  File "example.py", line 132, in surface_win32
    vkCreateWin32SurfaceKHR = vkGetInstanceProcAddr(instance, "vkCreateWin32SurfaceKHR")
  File "E:\Python\Python35\lib\site-packages\vulkan\__init__.py", line 5234, in vkGetInstanceProcAddr
    fn = ffi.cast('PFN_' + pName, fn)
ffi.error: undefined type name
PFN_vkCreateWin32SurfaceKHR
```

**Comment by realitix** [comment]
Hello @griffin2000, thanks for reporting this issue. I will fix it at the end of next week because I have to buy a new computer with Windows with Vulkan support. 
I will let you know.

**Comment by griffin2000** [comment]
Thanks. I may also take a look myself. Looking at the code it just seems that it just needs an extra wrapper function for vkCreateWin32SurfaceKHR added to _instance_ext_funcs, right?

**Comment by griffin2000** [comment]
I was able to fix this but hit another issue as one of the Win32 structure is not defined either

**Comment by griffin2000** [comment]
Just realized both this and https://github.com/realitix/vulkan/issues/7 are the caused by https://github.com/realitix/vulkan/blob/master/vulkan/_vulkan.py being generated on a non-windows platform. This means none of the windows symbols are defined. I guess the correct solution is to have one version of this file for each platform. How is it generated?

**Comment by realitix** [comment]
Fixed in 1.0.49 version
